src/client/renderer/r_state.c

```diff
@@ -917,6 +985,16 @@ void R_UseInterpolation(const vec_t lerp) {
 }
 
 /**
+ * @brief Uploads shell offset value to the currently loaded program.
+ */
+void R_UseShellOffset(const vec_t offset) {
+
+	if (r_state.active_program->UseShellOffset) {
+		r_state.active_program->UseShellOffset(offset);
+	}
+}
+
```

Comment:
Is there some way to "overload" glPolygonOffset for this, maybe? And if not, really, don't worry about it.